### PR TITLE
Body cells appear incorrectly/blank when trying to print ag-grid #95

### PR DESCRIPTION
**Pull request by mikhail-angelov**
wait for all new html nodes are loaded, before print document

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -40,16 +40,13 @@ class ReactToPrint extends React.Component {
     if (onBeforePrint) {
       onBeforePrint();
     }
+    target.contentWindow.focus();
+    target.contentWindow.print();
+    this.removeWindow(target);
 
-    setTimeout(() => {
```
Comment (MatthewHerbst) [inline src/index.js:44]: Why did you remove the `setTimeout` here? I'm not sure exactly why we needed it before, but it's been in the codebase almost since the first commit. @gregnb can you give any context on this please?

**Review by MatthewHerbst — COMMENTED** [review]
Thanks for the PR! I like what you've done here. I left one question to be addressed.

Also, now that DOM nodes will be going into `markLoaded` as well, the param there should probably be changed to just be `node`, and the error message should be updated to be more generic, since now a DOM node could fail to load, not just a link node.

File: src/index.js (hunk 1 above)
Comment (gregnb) [reply src/index.js:44]: hey @MatthewHerbst honestly I believe it because some browsers had issues if you tried to say close the window you were printing or in this case remove the element you are printing too quickly. Throwing in a hacky delay was how I found a lot of people to solve this issue:

https://stackoverflow.com/questions/6460630/close-window-automatically-after-printing-dialog-closes/19115657

if we don't think it's necessary anymore I'm ok with that but I would be just a little cautious here

Comment (MatthewHerbst) [reply src/index.js:44]: Thanks! @mikhail-angelov sounds like we should keep the `setTimeout` for now to be safe. I think it's unrelated to your core change anyways.

Comment (mikhail-angelov) [reply src/index.js:44]: Hi @MatthewHerbst you are right we should keep it
but actually my PR is not good enough, looks like my suggestion for root cause of this issue (sometime printed page is blank) is not correct, and I manage to reproduce the issue even with my fix, so I have to switch to print.js (it's more stable)
so, I close this PR, 
and probably make sense to investigate print.js and find difference between those 2 libs and make this lib more stable

**Comment by mikhail-angelov** [comment]
unfortunately, it does not solve the issue :(

**Comment by MatthewHerbst** [comment]
@mikhail-angelov sorry to hear that. If you have other thoughts on how to improve the library they would be greatly appreciated - a lot of us are trying to solve similar issues. Thanks!
